swagger.yml

```diff
@@ -535,6 +535,23 @@ definitions:
         type: number
         description: The longitude value of the location
         example: -73.61911979999999
+  
+  LocationSummary:
+    title: Location Summary
+    type: object
+    properties:
+      country:
+        type: string
+        description: The country
+        example: Canada
+      province:
+        type: string
+        description: The province
```

Comment:
Can we update the description to say `The province/state` (to account for US addresses)